package.json

```diff
@@ -9,7 +9,8 @@
   "scripts": {
     "watch": "webpack --config ./webpack.config.js --watch",
     "build": "webpack --config ./webpack.config.js",
-    "test": "karma start"
+    "test": "karma start",
+    "lint": "eslint normandy client webpack.config.js karam.conf.js"
```

Comment:
That makes sense. One reason I'm hesitant to do this is we rely on the exit code for CI. So if we do `|| true`, we can't use this for CI. I did some googling and I don't think there is any way to keep the exit code but not have npm lose its shit.

I think a better approach would be using [therapist](https://github.com/rehandall/therapist) instead of npm scripts to run lints, once that is ready.